lib/cfme/cloud_services/manifest_fetcher.rb

```diff
@@ -4,164 +4,28 @@ def self.fetch
     block_given? ? yield(manifest) : manifest
   end
 
+  def self.identity_headers
+    {"x-rh-identity" => Base64.strict_encode64(JSON.dump("identity" => {"account_number" => "cfme-tenant"}))}
+  end
+
   private_class_method def self.raw_manifest
-    # TODO: Download the manifest from the cloud for this CF version...for now just hardcode
-    #
     # TODO: Modeling
     #   - Should we allow collection of non-model information, such as replication,
     #     pg_* tables or filesystem level things?
-    common = {
-      "id"                  => nil,
-      "name"                => nil,
-      "type"                => nil,
-      "guid"                => nil,
-      "api_version"         => nil,
-      "emstype_description" => nil,
-      "hostname"            => nil,
-      "vms"                 => {
-        "id"                   => nil,
-        "name"                 => nil,
-        "type"                 => nil,
-        "guid"                 => nil,
-        "uid_ems"              => nil,
-        "archived"             => nil,
-        "cpu_cores_per_socket" => nil,
-        "cpu_total_cores"      => nil,
-        "disks_aligned"        => nil,
-        "ems_ref"              => nil,
-        "has_rdm_disk"         => nil,
-        "host_id"              => nil,
-        "linked_clone"         => nil,
-        "orphaned"             => nil,
-        "power_state"          => nil,
-        "ram_size_in_bytes"    => nil,
-        "retired"              => nil,
-        "v_datastore_path"     => nil,
-        "hardware"             => {
-          "id"    => nil,
-          "disks" => {
-            "id"           => nil,
-            "device_name"  => nil,
-            "device_type"  => nil,
-            "disk_type"    => nil,
-            "free_space"   => nil,
-            "mode"         => nil,
-            "size"         => nil,
-            "size_on_disk" => nil,
-            "partitions"   => {
-              "id"                => nil,
-              "name"              => nil,
-              "controller"        => nil,
-              "free_space"        => nil,
-              "partition_type"    => nil,
-              "size"              => nil,
-              "start_address"     => nil,
-              "used_space"        => nil,
-              "virtual_disk_file" => nil,
-              "volume_group"      => nil,
-              "volumes"           => {
-                "id"           => nil,
-                "name"         => nil,
-                "filesystem"   => nil,
-                "free_space"   => nil,
-                "size"         => nil,
-                "typ"          => nil,
-                "used_space"   => nil,
-                "volume_group" => nil,
-              }
-            }
-          }
-        }
-      }
-    }
+    uri_config = ::Settings[:cfme_cloud_services][:manifest_configuration]
```

Comment:
Minor but with Settings it's preferable to use dot notation... `Settings.cfme_cloud_service.manifest_configuration`